src/main/java/sg/ncl/MainController.java

```diff
@@ -1745,20 +1758,20 @@ public String editTeamProfile(
         JSONObject teamfields = new JSONObject();
         teamfields.put("id", teamId);
         teamfields.put("name", editTeam.getName());
-        teamfields.put("description", editTeam.getDescription());
-        teamfields.put("website", "http://default.com");
-        teamfields.put("organisationType", editTeam.getOrganisationType());
+        teamfields.put(DESCRIPTION, editTeam.getDescription());
+        teamfields.put(WEBSITE, "http://default.com");
+        teamfields.put(ORGANISATION_TYPE, editTeam.getOrganisationType());
         teamfields.put("privacy", "OPEN");
-        teamfields.put("status", editTeam.getStatus());
-        teamfields.put("members", editTeam.getMembersList());
+        teamfields.put(STATUS, editTeam.getStatus());
+        teamfields.put(MEMBERS, editTeam.getMembersList());
 
         HttpEntity<String> request = createHttpEntityWithBody(teamfields.toString());
         ResponseEntity response = restTemplate.exchange(properties.getTeamById(teamId), HttpMethod.PUT, request, String.class);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "response". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "response" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)